scripts/test_updates_all.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+V1_TAGS="0.1.0 0.2.0 0.3.0 0.4.0 0.4.1 0.4.2"
+V2_TAGS="0.5.0 0.6.0 0.6.1 0.7.0-pg9.6 0.7.1-pg9.6 0.8.0-pg9.6"
```

Comment:
add 0.1.0 to V2? worked before right?

Reply:
I added it, though I'm not sure I think it's for the best. It mostly works because we hadn't blocked things yet.